upi/openstack/03_bootstrap.yaml

```diff
@@ -25,15 +25,15 @@
     command:
       cmd: "openstack port set --tag {{ cluster_id_tag }} {{ os_port_bootstrap }}"
 
-  - name: 'Create the second bootstrap server port'
+  - name: 'Create the ACI containers bootstrap server port'
```

Comment:
Should be something like 'Create the bootstrap server opflex base port'
@snaiksat  can also confirm on this

Reply:
this one is fine (i was using opflex terminology before but i changed it)